### PR TITLE
Provide better copy for openshift/release Prow config format checker

### PR DESCRIPTION
**Pull request by petr-muller**

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
Can you prefix your comment with `[ERROR] ` on all those lines? It will get highlighted in the logs.

**Comment by petr-muller** [comment]
Sure, did not know that.

File: hack/validate-prow-job-ordering.sh [hunk 1]
```diff
@@ -18,4 +17,15 @@ cp -r "${jobs_dir}" "${workdir}"
...
 
-diff -Naupr "${jobs_dir}" "${workdir}/jobs"
+if ! diff -Naupr "${workdir}/jobs" "${jobs_dir}"> "${workdir}/diff"; then
+  cat << EOF
+This check enforces Prow Job configuration YAML file format (ordering,
+linebreaks, indentation) to be consistent over the whole repository. We have
+automation in place that manipulates these configs and consistent formatting
+helps reviewing the changes the automation does.
+
+Please apply the following changes to your Prow job configuration:
```
Comment (stevekuznetsov) [inline hack/validate-prow-job-ordering.sh]: You could also do something like

```
docker run -v ./ci-operator/jobs:/jobs registry.svc.ci.openshift.org/ci/determinize-prow-jobs:latest -- determinize-prow-jobs --prow-jobs-dir /jobs
```

**Pull request merged**